src/helpers/calculate.ratings.percentages.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * A fuction to calculate the percentage of ratings and to
+ * generate a structured object of data
+ * @param {Array} data - The array of the data to compute
+ * @returns {Array} data - The computed array
+ */
+export default (data) => {
+  // constructing the array to look like excpected
```

Comment:
@nkpremices Good work. I think we can improve this logic here. I'll think about it and get back to you.

Reply:
I'll be waiting @MemunaHaruna